packages/notification-center/src/i18n/languages/ku.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const KU: ITranslationEntry = {
+  translations: {
+    notifications: 'ئاگادارکردنەوەکان',
```

Comment:
This line gets translated to `Warnings` when reversing. Maybe it is better to find a more appropriate translation.